fhirflat/__init__.py

```diff
@@ -0,0 +1,14 @@
+from .resources.condition import Condition
```

Comment:
Can be simplified into one import by using a `__init__.py`

Reply:
Not sure what you mean here?

Reply:
so resources can be imported as `from .resources import Condition, Encounter, Patient ...`